### DISCUSSION TITLE
Gspro connection issue

### DISCUSSION THREAD

**Discussion by Wrightn13** [springbok/MLM2PRO-GSPro-Connector#94] [General]
I am getting the error that says gspro isn't running and to make sure it's running. I have made sure the path is correct to the gspro.exe file but it still won't connect. Any suggestions?

**Comment by RocknRollGolf** [comment]
You have the wrong file path. Go to GS pro core file and use the file path to the .exe file in there. I can send you the exact path later.

**Reply by RocknRollGolf** [reply]
That another issue that needs updating in the wiki

**Reply by Wrightn13** [reply]
The path I have is gsprov1/core/gsp/gspro.exe

**Reply by RocknRollGolf** [reply]
Yep that's it. That  should work

**Reply by Wrightn13** [reply]
That's what I thought. I even tried deleting everything and redoing and got the same issue. Any ideas what I should try to get it fixed?

**Reply by RocknRollGolf** [reply]
You can still open GS pro manually. 
Does the connector connect when you open it manually.

**Reply by Wrightn13** [reply]
That's the strange part gspro is open but when I hit connect it doesn't connect to it. I even tried turning off the auto open in the connector but that didn't work either.

**Comment by springbok** [comment]
Do you have the GSPro OpenAPI license, make sure the GSPro API window is open when you start GSPro. You can reset it by going into GSPro settings -> Game Settings -> Reset API window.

**Reply by Wrightn13** [reply]
That's the issue. I have the mevo plus license. I I'm guessing the only way it works is with the openapi license? I know I saw in another discussion but it there a way to keep the mevo plus connector but then use this one for putting?

**Reply by springbok** [reply]
No that's not possible, the connector communicates with GSPro via the OpenAPI so you need the OpenAPI license if you want to use the connector.

**Reply by RocknRollGolf** [reply]
No you have to go into gs pro and reset your license. It doesn't cost you anything you can just change it as many times as you want you can change it back to  flightscope licence anytime you like

**Reply by RocknRollGolf** [reply]
The only way to use the webcam putting is Springbok connector via the open API

**Reply by Wrightn13** [reply]
Okay thank you guys for the help. Hopefully I don't have anymore. Are there any videos or walk throughs on setting up the mevo plus to the connector? I've seen alot for the mlm2pro but haven't seen any on the mevoplus

**Reply by RocknRollGolf** [reply]
Here is the link for the lic converter. 
 https://gsprogolf.com/convert.html

**Reply by RocknRollGolf** [reply]
No there's no specific Mevo ones there's only the mlm pro one but it's very similar process. I am looking to do Mevo installation video soon. 
Try this link it for Mlm but similar process . Also read the wiki it has a Mevo section 

 https://youtu.be/-W3tzu48Ad0?si=qQPNYth3ooGcN3ql

**Reply by Wrightn13** [reply]
Okay thank you very much for the assistance.
